tests/test_inventory_service_interface.py

```diff
@@ -0,0 +1,59 @@
+from drift import inventory_service_interface
+import responses
+
+import unittest
+
+
+class InventoryServiceTests(unittest.TestCase):
+
+    HOST_TEMPLATE = '''
```

Comment:
Could this be moved out to a fixture file?

Reply:
done